### PR TITLE
[DOCS-7406] Update logs standard attribute doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

There's a new doc for all standard attributes. Updating the Logs standard attribute doc to direct to the new page and removing duplicate content. PM request.

[DOCS-7406](https://datadoghq.atlassian.net/browse/DOCS-7406)

Staging preview: https://docs-staging.datadoghq.com/may/update-logs-standard-attibutes/logs/log_configuration/attributes_naming_convention/

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-7406]: https://datadoghq.atlassian.net/browse/DOCS-7406?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/log_configuration/attributes_naming_convention.md [hunk 1]
```diff
@@ -301,24 +127,6 @@ See [Alias Facets][29] for additional information.
...
-[21]: /tracing/app_analytics/search/
-[22]: /integrations/rsyslog/
-[23]: /integrations/nxlog/
-[24]: /integrations/syslog_ng/
-[25]: /integrations/fluentd/
-[26]: /integrations/logstash/
-[27]: https://en.wikipedia.org/wiki/List_of_DNS_record_types
-[28]: /logs/explorer/facets/#aliased-facets
-[29]: /logs/explorer/facets/#alias-facets
+[9]: https://docs.datadoghq.com/standard-attributes/?product=log+management
```
Comment (brett0000FF) [inline content/en/logs/log_configuration/attributes_naming_convention.md:130]:
```suggestion
[9]: /standard-attributes/?product=log+management
```

**Review by brett0000FF — APPROVED** [review]
Nice! Looks good. Just a small suggestion to use the relative doc link for reference link `[9]`.

Glad we are converging content to the new standard attribute page!

File: content/en/logs/log_configuration/attributes_naming_convention.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ The standard attribute panel appears when you add a new standard attribute or ed
 
 ### Default standard attribute list
 
-The default standard attribute list is split into the functional domains:
+See the full list of [Log Management Default Standard Attributes][9], which is split into the functional domains:
 
 - [Network/communications](#web-access)
```
Comment (LucasC) [inline content/en/logs/log_configuration/attributes_naming_convention.md:88]: @maycmlee We need to update all these shortlinks as well.
`#web-access` should redirect instead to `https://docs-staging.datadoghq.com/may/update-logs-standard-attibutes/standard-attributes/?product=log+management&search=network.` (notice the search query done to filter by prefix)

**Review by LucasC — CHANGES_REQUESTED** [review]

File: content/en/logs/log_configuration/attributes_naming_convention.md (hunk 2 above)
Comment (maycmlee) [reply content/en/logs/log_configuration/attributes_naming_convention.md]: @LucasC good catch. I added the links and also pulled back in some of the information we had for each section. Let me know know what you think, thanks!

**Review by LucasC — APPROVED** [review]

**Pull request merged**